### PR TITLE
Prebuilt filter fix

### PR DESCRIPTION
**Pull request by TheSlimvReal**
The prebuilt filters directly work on the config and currently this changes are also exported when downloading the config. This has some bad sideeffects when using this config again. Instead the exported config is now directly taken from the database.

### Visible/Frontend Changes
- [x] Prebuilt schools-filter is always present (not only after second entry of the page #761

### Architectural/Backend Changes
- [x] Loading config from database before exporting
- [x] Retrigger `ngOnChanges` of entity list component after schools-filter is built

### CODE REVIEW

File: src/app/core/config/config.service.spec.ts [hunk 1]
```diff
@@ -30,13 +31,15 @@ describe("ConfigService", () => {
   }));
 
   it("should use the default config when none is loaded", fakeAsync(() => {
-    const configBefore = service.getAllConfigs("");
-    entityMapper.load.and.throwError("No config found");
+    const defaultConfig = Object.keys(defaultJsonConfig).map((key) => {
+      defaultJsonConfig[key]._id = key;
```
Comment (sleidig) [inline src/app/core/config/config.service.spec.ts:35]: why should there be these _id on each config item? They are not present on the config we usually have so far.

**Review by sleidig — COMMENTED** [review]
fixes the bug :+1: 

I don't understand the additional `_id`s on the config export, however.

File: src/app/core/config/config.service.spec.ts (hunk 1 above)
Comment (TheSlimvReal) [reply src/app/core/config/config.service.spec.ts:35]: This is due to the fact that we set the `_id` field when using `getAllConfigs` [here](https://github.com/Aam-Digital/ndb-core/blob/master/src/app/core/config/config.service.ts#L61).
In this case I *fake* the side effect of the `getAllConfigs` function

Comment (sleidig) [reply src/app/core/config/config.service.spec.ts:35]: ok, that export of config previously already contained the _id properties. nevermind

**Review by sleidig — APPROVED** [review]

**Pull request merged**